frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -65,6 +65,7 @@ export enum FLAGS {
   CAN_LIST_NODE = 'CAN_LIST_NODE',
   CAN_LIST_PV = 'CAN_LIST_PV',
   CAN_LIST_CRD = 'CAN_LIST_CRD',
+  CAN_LIST_CLUSTER_QUOTA = 'CAN_LIST_CLUSTER_QUOTA',
```

Comment:
Instead of adding a flag, I would just do an access review in the component. The `CAN_LIST_FOO` flags are typically needed to hide nav items, but not for general access reviews.